dev_container

```diff
@@ -475,6 +475,11 @@ launch() {
         conditional_opt+=" -v /opt/yuan/qcap/lib:/opt/yuan/qcap/lib"
     fi
 
+    # add tegra directory
+    if [ -d  /usr/lib/aarch64-linux-gnu/tegra ]; then
```

Comment:
Which libraries are we depending on from here, are they not mounted with `runtime=nvidida`?

Reply:
These are the multimedia libraries

Reply:
Is this an issue on dGPU only? the nvidia runtime should handle things

Reply:
the nvidia runtime does not add this directory